vespalib/src/vespa/vespalib/io/fileutil.cpp

```diff
@@ -398,18 +398,13 @@ File::sync()
 {
     if (_fd != -1) {
         if (::fsync(_fd) == 0) {
-            LOG(debug, "sync(%s): File synchronized with disk.",
-                _filename.c_str());
+            LOG(debug, "sync(%s): File synchronized with disk.",_filename.c_str());
         } else {
-            asciistream ost;
-            ost << "sync(" << _filename << "): Failed, errno(" << errno << "): "
-                << safeStrerror(errno);
-            throw IoException(ost.str(), IoException::getErrorType(errno),
-                              VESPA_STRLOC);
+            LOG(warning, "fsync(%s): Failed to sync file. errno(%d): %s",
+                _filename.c_str(), errno, safeStrerror(errno).c_str());
```

Comment:
Should we use a buffering logger here (`LOGBP` and similar) to avoid spamming the log? If `fsync` fails once due to the underlying fs not supporting it, it's likely to fail _every_ time

Reply:
I doubt that fsync is the root cause. It reports "#define EACCES          13      /* Permission denied */" which is not listed as a possible errno for fsync.
I hope that this might cause it to progress and perhaps shed some light on what is the root cause.

What is LOGBP using for duplicate detection ? Call site or actual message ?

Reply:
From what I can recall `LOGBP` uses the call site (`__FILE__` + `__LINE__` combo), so no need to provide an explicit dedupe token